evap/evaluation/tests/test_models.py

```diff
@@ -1139,3 +1140,37 @@ class QuestionnaireTests(TestCase):
     def test_locked_contributor_questionnaire(self):
         questionnaire = baker.prepare(Questionnaire, is_locked=True, type=Questionnaire.Type.CONTRIBUTOR)
         self.assertRaises(ValidationError, questionnaire.clean)
+
+
+class QuestionTests(TestCase):
+    def test_save_for_text_and_heading_question_type(self):
+        questionaire = baker.make(Questionnaire)
+        question_text = baker.prepare(
+            Question,
+            questionnaire=questionaire,
+            type=QuestionType.TEXT,
+            allows_additional_textanswers=True,
+            counts_for_grade=True,
+        )
+        question_heading = baker.prepare(
+            Question,
+            questionnaire=questionaire,
+            type=QuestionType.HEADING,
+            allows_additional_textanswers=True,
+            counts_for_grade=True,
+        )
+
+        with patch.object(models.Model, "save") as mock_save:
+            question_text.save(update_fields=["text_de"])
+            mock_save.assert_called_once()
+            args, kwargs = mock_save.call_args
+            self.assertEqual(
+                set(kwargs["update_fields"]), {"allows_additional_textanswers", "counts_for_grade", "text_de"}
+            )
+
+            question_heading.save(update_fields=["text_de"])
+            self.assertEqual(mock_save.call_count, 2)
```

Comment:
There is [`reset_mock`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.reset_mock), which would probably make sense here (to assert two times that there was exactly the one call you expected)